curvlinops/kfac.py

```diff
@@ -188,6 +189,9 @@ def __init__(
                 language modeling. If ``None``, the loss function is a sum. This
                 argument is used to ensure that the preconditioner is scaled
                 consistently with the loss and the gradient. Default: ``"batch"``.
+            num_per_example_loss_terms: Number of per-example loss terms. If ``None``,
+                it is inferred from the data at the cost of one traversal through
```

Comment:
Can we add a complementary explanation how this number is related to the dimensionality of the random variable whose Fisher we're KFAC-approximating?

Reply:
Please check if this is sufficient.